src/RadioGroup/RadioElement.tsx

```diff
@@ -22,8 +23,13 @@ export const RadioElement: FC<RadioElementProps> = ({
   value,
   checked,
   onChange,
+  isError,
   ...marginProps
 }) => {
+  console.log({
```

Comment:
Is there a reason this has been left in?

Reply:
Nop, forgot about it, will remove thanks!